src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -918,8 +918,12 @@ export class SearchSource {
     };
     body.query = buildEsQuery(index, query, filters, esQueryConfigs);
 
-    // For testing shard failure messages in UI, uncomment the next block and switch to `kibana*` data view
-    // body.query = {
+    // For testing shard failure messages in the UI, follow these steps:
+    // 1. Add all three sample data sets (flights, ecommerce, logs) to Kibana.
+    // 2. Create a data view using the index pattern `kibana*`.
```

Comment:
Adding a note how to include all indices in the request, since else it depends on which timestamp is selected
```suggestion
    // 2. Create a data view using the index pattern `kibana*` and don't use a timestamp field.
```

Reply:
Odd I think we actually used a timestamp field and it worked, but I don't think it would hurt to add this too.